test/extended/router/idle.go

```diff
@@ -101,6 +101,9 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 			o.Expect(err).NotTo(o.HaveOccurred(), "failed to fetch the service")
 			mustVerifyIdleAnnotationValues(annotations)
 
+			// wait for target deployment to actually scale down
+			err = waitForRunningPods(oc, 0, exutil.ParseLabelsOrDie("app=idle-test"), timeout)
```

Comment:
Add a check that we didn't get an errror. `o.Expect(err).NotTo(o.HaveOccurred())`.

Reply:
fixed.